Actions/PrimaryGeneratorAction.cpp

```diff
@@ -65,17 +65,19 @@ void PrimaryGeneratorAction::GeneratePrimaries(G4Event* event)
       fPrimaryGenerator->GenerateEvtLargeChamber(event);
     } else {
       G4Exception("PrimaryGeneratorAction", "PG05", FatalException,
-                  "Called run with non-exisitng geometry");
+                  "Called run with non-exisitng geometry 1");
```

Comment:
What does "1" (and later "2") mean in this case? Could you make the error messages more descriptive?

Reply:
There are two same messages in this class, so that was a quick way to know what is causing the problem. I made them more clear.